docs/en/guides/webhooks/batch-deliveries.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: default
+title: Webhooks Batch Deliveries
+pagename: webhooks-batch-deliveries
+lang: en
+---
+
+# Batch Delivery
+
+## Enabling Batch Delivery
+
+You can choose to configure your webhook so that it will make requests with a batched payload format, by selecting the "Deliver in Batched Format" option.
```

Comment:
```suggestion
You can choose to configure your webhook so that it will make requests with a batched payload format by selecting the "Deliver in Batched Format" option.
```